docs/nodejs/angular-tutorial.md

```diff
@@ -164,7 +164,7 @@ If you'd like to see a full MEAN (MongoDB, Express, Angular, Node.js) stack exam
 
 ## Angular Extensions
 
-In addition to the functionality, VS Code provides out of the box, you can install VS Code extensions for greater functionality.
+In addition to the features VS Code provides out of the box, VS Code extensions can be installed for greater functionality.
```

Comment:
Thanks for fixing the first part of this sentence (weird comma placement).
I'd rather leave the second part of the sentence as is and not convert to passive voice.

Reply:
np. I've reverted the part after the comma.